gsa/src/web/components/icon/svgicon.js

```diff
@@ -56,6 +56,34 @@ const Styled = styled.span`
   }
 `;
 
+const useIsMountedRef = () => {
+  const ref = useRef();
+  ref.current = true;
+  // if the ref changes, which is the case when the component unmounts
+  // set the ref.current to false in order to prevent state updates in
+  // useStateWithRefCheck()
```

Comment:
```suggestion
  // useStateWithMountCheck()
```

Reply:
Oops... I'm going to include this in the coming PR update.